### PR TITLE
Update saas-digital-business-journey-azure.md

### PR DESCRIPTION
**Pull request by nickward13**
Added in details for Founders Hub and the core startup stack architecture

### CODE REVIEW

**Review by mekenthompson — APPROVED** [review]

**Review by mekenthompson — APPROVED** [review]
Ack

**Comment by mekenthompson** [comment]
@nickward13 Unsurprisingly I no longer have write access to properly approve this :) - Maybe you could reach out to Docs team to take over ownership of this?

**Comment by Court72** [comment]
@EdPrice-MSFT 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by EdPrice-MSFT — DISMISSED** [review]
Looks good!

**Comment by EdPrice-MSFT** [comment]
#Sign-Off

**Review by EdPrice-MSFT — COMMENTED** [review]
Looks fantastic! Thank you for adding this!

**Comment by EdPrice-MSFT** [comment]
#Sign-Off

**Comment by Jak-MS** [comment]
closing / reopening to see if this will clear

**Comment by nickward13** [comment]
#Sign-Off

**Comment by nickward13** [comment]
Thanks all - I've signed the CLA.  Please let me know if there is anything else I need to do to help merge.

File: docs/guide/saas/saas-digital-business-journey-azure.md [hunk 1]
```diff
@@ -24,8 +24,9 @@ Follow this guidance to get the basics right when you build your software as a s
 ## Evaluate Azure
 
 Get informed and make the right platform and partnership decisions for your SaaS business.
-
-- Evaluate these [Azure service architecture options](../../browse/index.yml) to find the right real-world Azure architecture for your service.
+- If you are a startup, join [Microsoft Founders Hub](https://www.microsoft.com/startups) to get free access to the Microsoft cloud, to build your product (via Azure, GitHub, or Visual Studio) and to run your business (via Microsoft 365 or Dynamics 365).
```
Comment (Court72) [inline docs/guide/saas/saas-digital-business-journey-azure.md:27]:
```suggestion
- If you are a startup, join [Microsoft Founders Hub](https://www.microsoft.com/startups) to get free access to the Microsoft cloud, to build your product (via Azure, GitHub, or Visual Studio), and to run your business (via Microsoft 365 or Dynamics 365).
```

**Pull request merged**